src/test/scala-2.11/markets/clearing/engines/CDAMatchingEngineSpec.scala

```diff
@@ -464,19 +463,19 @@ class CDAMatchingEngineSpec extends TestKit(ActorSystem("CDAMatchingEngineSpec")
       val askPrice = randomLong(prng, upper = bidPrice)
       val askQuantity = randomLong(prng, marketBidQuantity, limitBidQuantity)
       val askOrder = LimitAskOrder(askOrderIssuer, askPrice, askQuantity, randomLong(prng), testTradable, uuid)
-      val fills = matchingEngine.findMatch(askOrder)
+      val matchings = matchingEngine.findMatch(askOrder)
 
-      Then("the matching engine should generate a PartialMatch at the best limit price.")
+      Then("the matching engine should generate a Matching at the best limit price.")
 
       val residualAskQuantity = askQuantity - marketBidQuantity
-      val (filledAskOrder, residualAskOrder) = askOrder.split(residualAskQuantity)
-      val partialFill = PartialMatch(marketBidOrder, filledAskOrder, bidPrice)
+      val (_, residualAskOrder) = askOrder.split(residualAskQuantity)
+      val partialFill = Matching(askOrder, marketBidOrder, Some(bidPrice), Some(residualAskOrder), None)
 
       val residualBidQuantity = limitBidQuantity - residualAskQuantity
-      val (filledBidOrder, residualBidOrder) = limitBidOrder.split(residualBidQuantity)
-      val totalFill = TotalMatch(filledBidOrder, residualAskOrder, bidPrice)
+      val (_, residualBidOrder) = limitBidOrder.split(residualBidQuantity)
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Field name does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1545342729/issues/source?bid=2921361&fileBranchId=3015559#l475)